docs/docs/deploying-to-now.md

```diff
@@ -1,80 +1,31 @@
 ---
-title: Deploying to Now
+title: Deploying to Vercel Now
 ---
 
-[ZEIT Now](https://zeit.co/now) is a [cloud platform for serverless deployment](https://zeit.co/docs/v2/getting-started/introduction-to-now/) that you can use to deploy your Gatsby projects and [alias them](https://zeit.co/docs/v2/domains-and-aliases/aliasing-a-deployment/) to your personal domain or a free `.now.sh` suffixed URL.
+[Vercel Now](https://vercel.com/now) is a cloud platform for websites and serverless APIs, that you can use to deploy your Gatsby projects to your personal domain (or a free `.now.sh` suffixed URL).
 
 This guide will show you how to get started in a few quick steps:
 
 ## Step 1: Getting Started with Gatsby
 
-If you haven't already [set up a Gatsby project](https://www.gatsbyjs.org/docs/quick-start) you can do so by first installing Gatsby globally:
-
-```shell
-npm install --global gatsby-cli
-```
-
-Then generate a project with the following command:
-
-```shell
-npx gatsby new <your project name>
-```
-
-## Step 2: Getting Now
-
-You can use Now by installing [Now Desktop](https://zeit.co/docs/v2/getting-started/installation/#now-desktop), which also installs Now CLI and keeps it up-to-date automatically.
-
-To install Now CLI quickly with npm, use the following:
+To install the Vercel command-line interface with [npm](https://www.npmjs.com/), run the following command:
 
 ```shell
 npm install -g now
 ```
 
-## Step 3: Preparing to Deploy
+## Step 2: Deploying
 
-With Now CLI installed, we can go on to deploy our previously setup Gatsby project by first creating a `now.json` file with the following contents:
-
-```json:title=now.json
-{
-  "version": 2,
-  "name": "my-gatsby-project",
-  "builds": [
-    {
-      "src": "package.json",
-      "use": "@now/static-build",
-      "config": { "distDir": "public" }
-    }
-  ]
-}
-```
-
-This `now.json` file will allow us to do several things, specifically:
-
-- Use the [latest Now 2.0 version](https://zeit.co/blog/now-2) of [the platform](https://zeit.co/docs/v2/platform/overview/)
-- Set the project name to `my-gatsby-project`
-- Use the [@now/static-build builder](https://zeit.co/docs/v2/deployments/official-builders/static-build-now-static-build/) to take the `package.json` file as an entrypoint and use the `public` directory as our content directory
-
-The final step is to add a script to the `package.json` which will build our application:
-
-```json:title=package.json
-{
-  "scripts": {
-    ...
-    "now-build": "npm run build"
-  }
-}
-```
-
-## Step 4: Deploying
-
-You can deploy your application by running the following in the root of the project directory, where the `now.json` is:
+You can deploy your application by running the following command in the root of the project directory:
```

Comment:
Is the now.json file no longer needed?

Reply:
I just built a Gatsby project and deployed it to Vercel Now. It doesn't seem like a now.json file is necessary, instead, it creates a `.now` folder with `project.json`. This is auto-generated though, so seems to be alright.

Reply:
@lannonbr  The `now.json` file is no longer necessary as you can deploy directly for your Vercel dashboard, but in some scenarios, developers would want to initiate deployment once they push to a GitHub branch, and if Vercel is already installed in every repo once the `now.json` is added to a branch it automatically deploys that branch to now without heading to your Vercel dashboard. 

That's one major application why I feel we should still keep it, Vercel still supports it and will continue supporting it so we should too, it's a great feature. I do not deploy from the dashboard, I'll rather write a `now.json` file and auto-deploy on the push to a branch rather than go to a dashboard :)

Reply:
Yes @AmruthPillai, you could also deploy from the terminal using the `now --prod command, that's another thing, but then we know the `.now` the branch is under `nowignore` so it does not get pushed. 

So it's more of Gatsby's choice whether to retain this or dissolve this cc @Ekwuno

Reply:
This is the existing doc on deploying to Zeit. https://www.gatsbyjs.org/docs/deploying-to-zeit-now/ Ayo if you update your fork of the repo you should see the updates on your local.

Reply:
Yes, I would do that before the day runs out.